README.md

```diff
@@ -37,9 +35,36 @@ multiple Terraform modules, remote state, and locking.
    Terraform you already have installed. However, based on the settings in your `terraform.tfvars` file, Terragrunt can
    configure remote state, locking, extra arguments, and lots more.
 
+1. For some optional, yet useful, background reading that will help explain the motivation for Terragrunt, see: [How to create reusable infrastructure with Terraform modules](https://blog.gruntwork.io/how-to-create-reusable-infrastructure-with-terraform-modules-25526d65f73d) and [How to use Terraform as a team](https://blog.gruntwork.io/how-to-use-terraform-as-a-team-251bc1104973).
```

Comment:
Consider adding that Terragrunt is also a direct implementation of the ideas expressed in terraformupandrunning.com.